mesecons_slippyblocks/init.lua

```diff
@@ -0,0 +1,76 @@
+-- Stippy blocks can be used together with pistons or movestones
+	
+	minetest.register_node("mesecons_slippyblocks:slippy_block", {
+	description = "Slippy block)",
+	drawtype = "nodebox",
+	tiles = {"mesecons_slippyblocks.png"},
+	walkable     = true,
+	pointable    = true,
+	diggable     = true,
+	buildable_to = true,
+	groups = {cracky=3,oddly_breakable_by_hand=3},
```

Comment:
This would look nicer:
```suggestion
	groups = {cracky = 3, oddly_breakable_by_hand = 3},
```
(Not only here.)